easysnmp/interface.c

```diff
@@ -3788,6 +3823,10 @@ static PyObject *netsnmp_set(PyObject *self, PyObject *args)
     {
         if (!PyArg_ParseTuple(args, "OO", &session, &varlist))
         {
+            error = 1;
+            PyErr_SetString(PyExc_ValueError,
```

Comment:
This can be condensed to a single line:
```
            PyErr_SetString(PyExc_ValueError,
                    "missing 2 required arguments: 'session' and 'varlist'");
```

Reply:
ok